src/utils/innerSliderUtils.js

```diff
@@ -565,9 +567,12 @@ export const getTrackCSS = spec => {
     "slideWidth"
   ]);
   let trackWidth, trackHeight;
+  const variableWidthMax = 5000; // Arbitrary width, mirrors what is used by Slick Carousel
   const trackChildren = spec.slideCount + 2 * spec.slidesToShow;
-  if (!spec.vertical) {
+  if (!spec.vertical && !spec.variableWidth) {
     trackWidth = getTotalSlides(spec) * spec.slideWidth;
+  } else if (!spec.vertical && spec.variableWidth) {
+    trackWidth = trackChildren * variableWidthMax;
```

Comment:
This section is the only part I changed. Aside from Jeseeq's fix above, everything else was linter autocorrects.